### PR TITLE
LEAF-2090: search/add employee enhancement

### PR DESCRIPTION
**Pull request by raphaelfretter**

### CODE REVIEW

**Review by AmandaSumner — APPROVED** [review]

**Review by maxrichardjr — APPROVED** [review]
Looks good

**Review by gerald-todd — APPROVED** [review]
Looks a lot better :)

**Review by Pelentan — APPROVED** [review]

**Review by gsteinhoff — APPROVED** [review]

File: LEAF_Nexus/js/groupSelector.js [hunk 1]
```diff
@@ -5,231 +5,300 @@
...
+                url: this.apiPath + "group/search",
+                dataType: "json",
+                data: { q: this.q, tag: this.tag, noLimit: this.optionNoLimit },
+                success: function (response) {
+                    t.currRequest = null;
+                    t.selection = "";
+                    t.numResults = 0;
+                    t.jsonResponse = response;
+
+                    console.log(response);
```
Comment (aerinkayne) [inline LEAF_Nexus/js/groupSelector.js:201]: console log

**Review by aerinkayne — COMMENTED** [review]
Just the one console log, but otherwise looks good.

File: LEAF_Nexus/js/groupSelector.js (hunk 1 above)
Comment (raphaelfretter) [reply LEAF_Nexus/js/groupSelector.js]: console log removed

**Pull request merged**